modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportEntitiesWithPartiallyUpdate.testcase

```diff
@@ -47,6 +47,7 @@ definition {
 				fieldMappings = "articleBody:articleBody,headline:headline,externalReferenceCode:externalReferenceCode",
 				fileName = "json_blogPosting_import.json",
 				importStrategy = "Only Add New Records",
+				scope = "Liferay DXP",
```

Comment:
I am wondering if we need to add something like test.run.environment = "EE" property? If we are run the case on CE test suite, the scope should be `Liferay` instead of `Liferay DXP`. We can see if relevant passed or not first. @magjed4289 any ideas?

Reply:
With the test failures, I have checked locally, if we don't use `'test.liferay.virtual.instance = "false";'`, it will run in a virtual instance and the option into the scope will change to 'liferay-v3j3q8g6', if we use it, it will run with 'Liferay DXP'
cc: @magjed4289 @yanan

Reply:
@yanan @Hijky The string that is repeatable in all the instance names is "liferay", so maybe we can use this to map the scope names as `contains "liferay"` along the tests? The other one name is always Global, correct?

Reply:
@magjed4289 @yanan I found the option name is the same as the site name and I used the getSiteName(TestCase.getSiteName();) method to pass the scope value